include/mbedtls/config.h

```diff
@@ -56,6 +56,31 @@
 #define MBEDTLS_HAVE_ASM
 
 /**
+ * \def MBEDTLS_NO_UDBL_DIVISION
+ *
+ * The platform lacks support for double-width integer division (64-bit
+ * division on a 32-bit platform, 128-bit division on a 64-bit platform).
+ *
+ * Used in:
+ *      include/mbedtls/bignum.h
+ *      library/bignum.c
+ *
+ * The bignum code uses double-width division to speed up some operations.
+ * Double-width division is often implemented in software that needs to
+ * be linked with the program. The presence of a double-width integer
+ * type is usually detected automatically through preprocessor macros,
+ * but the automatic detection cannot know whether the code needs to
+ * and can be linked with an implementation of division for that type.
+ * By default division is assumed to be usable if the type is present.
+ * Uncomment this option to prevent the use of double-width division.
+ *
```

Comment:
I still think that the beginning of the description should be changed to not say that the option means that the platform lacks support for double-width types, especially now that a reason for disabling their use is given even if the platform supports it.

Also, I'd prefer to change the exact wording regarding the presence of division here, because it sounds as if we were referring to the standard when saying that multiplication / division need not be supported. What about something like "Furthermore, while the standard requires a 64-bit type even on a 32-bit platform, some implementations do not adhere to that and omit support for multiplication and division."?